{{ cookiecutter.project_slug }}/docker/Dockerfile.release

```diff
@@ -0,0 +1,6 @@
+FROM {{ cookiecutter.project_slug }}.base
```

Comment:
If following Makefile, it should be `FROM {{cookiecutter.project_slug}}-image-base`.

Reply:
👍